### ISSUE TITLE
CelebaHQ manual data preparation no longer possible?

### ISSUE THREAD

**Issue by zplizzi** [tensorflow/datasets#1496]
The docs [here](https://www.tensorflow.org/datasets/catalog/celeb_a_hq) suggest that you need a set of `.tar` files in order to use the CelebaHQ dataset, but the [linked repository](https://github.com/tkarras/progressive_growing_of_gans) for generating them now has what appears to be a different format for the generated dataset, with `.tfrecords` files. I dug around in the history of that repo but couldn't find instructions for the old way of generating `.tar` files. If someone could link to a way to generate these old-style files it would be much appreciated, otherwise I think this dataset loader needs updated.

**Comment by dhirensr** [comment]
@zplizzi : I agree that the mentioned repo doesn't has any links to download the .tar file, however I think this link would help you https://drive.google.com/drive/folders/1jU-hzyvDZNn_M3ucuvs9xxtJNc9bPLGJ to download celeba 1024x1024 images.

**Comment by Conchylicultor** [comment]
Thanks for catching this. It seems the processing is now done with https://github.com/tkarras/progressive_growing_of_gans/blob/master/dataset_tool.py#L454 but not sure where the data is coming from.
If you found where they moved file (e.g. looking at the README history, or even open a bug in their repo.), don't hesitate to send a PR to update the dataset.

**Comment by dhirensr** [comment]
@Conchylicultor : I tried to find the links however was not sucessful. However I found the drive link where one can download the celebahq 1024x1024 images. (https://drive.google.com/drive/folders/1jU-hzyvDZNn_M3ucuvs9xxtJNc9bPLGJ)
so maybe if we are not able to find the link, then we can change the processing code of the dataset to adapt to them.

**Comment by vijayphoenix** [comment]
Hi @Conchylicultor @dhirensr @zplizzi 
I don't think a lot of changes will be required if we use the .zip files from this [drive link](https://drive.google.com/open?id=11Vz0fqHS2rXDb5pprgTjpD7S2BAJhi1P).

Thank you

**Comment by moflo** [comment]
Hi - there is an s3 mirror of CelebA images available and I wrote a TFDS loader for it:
https://github.com/moflo/celebalocal
